test/unit/uk/gov/hmrc/exports/models/declaration/ExportsDeclarationSpec.scala

```diff
@@ -155,7 +155,7 @@ object ExportsDeclarationSpec {
       |    },
       |    "officeOfExit": {
       |      "officeId": "GB000054",
-      |      "circumstancesCode": "No"
+      |      "answer": "Yes"
```

Comment:
nit: change probably not required

Reply:
Probably makes more sense as `circumstancesCode` is not longer used in our FE model/pages.